content/guides/references/migration-guide.md

```diff
@@ -83,6 +83,164 @@ on('<event>', (arg1, arg2) => {
 
 CONTENT_TBD
 
+## Migrating `cypress-file-upload` to [`.selectFile()`](/api/commands/selectfile)
+
+Selecting files with input elements or dropping them over the page is available
+in Cypress 9.3. Read the [`.selectFile()` API docs](/api/commands/selectfile)
+for more information on how this works and how to use it. This guide details how
+to change your test code to migrate from the
+[`cypress-file-upload`](https://github.com/abramenal/cypress-file-upload) plugin
+to `.selectFile()`.
+
+#### Quick guide
```

Comment:
I feel like all `####` headers in this PR can be changed to `###`, and the `Examples` section header can possibly go away. This would make the right-side nav look more consistent with that of other migration guide sections. Thoughts?

Reply:
I'm changing quick guide to ###, but I like Examples as a ### because then it shows up in the table of contents, but each individual example doesn't. Since only 2nd and 3rd level headers are rendered:

![image](https://user-images.githubusercontent.com/3003404/148590245-6c98c27a-c50d-4715-aab1-9d798dbe15da.png)

A TOC that looks like

## Migrating from `cypress-file-upload` to [`.selectFile()`](/api/commands/selectfile)
### Quick Guide
### Examples

looks pretty clean while this one is overly verbose for the right-hand column TOC:

## Migrating from `cypress-file-upload` to [`.selectFile()`](/api/commands/selectfile)
### Quick Guide
### Read and attach a fixture
### Using drag-n-drop
### Overriding the file name
### Working with file contents